cmd/stunc/stunc.go

```diff
@@ -5,24 +5,40 @@
 package main
 
 import (
+	"flag"
 	"log"
 	"net"
 	"os"
 	"strconv"
+	"time"
 
 	"tailscale.com/net/stun"
 )
 
 func main() {
 	log.SetFlags(0)
-
-	if len(os.Args) < 2 || len(os.Args) > 3 {
-		log.Fatalf("usage: %s <hostname> [port]", os.Args[0])
-	}
-	host := os.Args[1]
+	var host string
 	port := "3478"
-	if len(os.Args) == 3 {
-		port = os.Args[2]
+
+	var readTimeout time.Duration
+	flag.DurationVar(&readTimeout, "timeout", 30*time.Second, "response wait timeout")
+
+	flag.Parse()
+
+	values := flag.Args()
+	if host == "" && (len(values) < 1 || len(values) > 2) {
```

Comment:
`host == ""` is always true

Reply:
Ah this is leftover from another iteration, thanks!